src/azure-cli-core/azure/cli/core/commands/template_create.py

```diff
@@ -7,9 +7,9 @@
 import argparse
 import platform
 
-from azure.cli._util import CLIError
-from azure.cli.commands import register_cli_argument
-from azure.cli.commands.arm import (
+from azure.cli.core._util import CLIError
+from azure.cli.core.commands import register_cli_argument
+from azure.cli.core.commands.arm import (
```

Comment:
> azure.cli.core.commands.arm [](start = 4, length = 28)

Should ARM be part of core? (I'd say yes but I think it's worth asking the question).